app/modules/transcribe/transcribe.controller.js

```diff
@@ -45,6 +42,23 @@ function TranscribeController($stateParams, $modal, $scope, $window, Annotations
             });
     }
 
+    function variantsFeedback() {
+        var annotations = AnnotationsFactory.list();
+        var vArray = []
+        annotations.forEach(function (el) {
+            if (el.type === 'text' && el.variants) {
+                vArray.push(el.variants);
+            }
+        });
+        if (vArray) {
```

Comment:
This is always going to be true, as an empty array is a truthy value in Javascript :trollface: Use `(vArray.length)` instead, as `0` is always falsy.